# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/unit/xpcshell.ini
# Commit: 039e8fd2928d
# Full Hash: 039e8fd2928d8d368ad892fb065bb2471e13d86e
# Author: Boris Zbarsky <bzbarsky@mit.edu>
# Date: 2020-02-22 09:52:08
# Regressor Bug: 267645
# File Overlap Count: 1
# Description:
#   Bug 267645 part 2.  Sanitize exceptions from exported functions. r=xeonchen,bholley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35996
# ==============================================================================

diff -r 942a92874526 -r 039e8fd2928d js/xpconnect/tests/unit/xpcshell.ini
--- a/js/xpconnect/tests/unit/xpcshell.ini	Sat Feb 22 04:56:52 2020 +0000
+++ b/js/xpconnect/tests/unit/xpcshell.ini	Sat Feb 22 05:01:59 2020 +0000
@@ -23,6 +23,7 @@
 
 [test_allowWaivers.js]
 [test_bogus_files.js]
+[test_bug267645.js]
 [test_bug408412.js]
 [test_bug451678.js]
 [test_bug604362.js]
